src/ApiRequestPanel.js

```diff
@@ -130,13 +131,14 @@ export class ApiRequestPanel extends EventsTargetMixin(HeadersParserMixin(LitEle
       .scope="${aware}"
       @api-changed="${this._apiChanged}"></raml-aware>` : ''}
 
+    ${this._renderServerSelector()}
     <api-request-editor
       ?narrow="${narrow}"
       .redirectUri="${redirectUri}"
       .selected="${selected}"
       .amf="${amf}"
       ?noUrlEditor="${noUrlEditor}"
-      .baseUri="${baseUri}"
+      .baseUri="${selectedServer}"
```

Comment:
This is not enough. Actually you shouldn't be doing it this way. As we agreed today, you need to compute the model for the server and pass it to the editor. It already handles [server property](https://github.com/advanced-rest-client/api-request-editor/blob/stage/src/ApiRequestEditor.js#L94) made for the partial model. The base URI stays as it was.